test/cpp/lite_interpreter/test_cores.cpp

```diff
@@ -0,0 +1,54 @@
+#include <gtest/gtest.h>
+#include <torch/csrc/autograd/generated/variable_factories.h>
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/frontend/resolver.h>
+#include <torch/csrc/jit/mobile/import.h>
+#include <torch/csrc/jit/mobile/module.h>
+#include <torch/torch.h>
+
+#include <unordered_set>
+
+namespace torch {
+namespace jit {
+namespace mobile {
+
+TEST(CoresTest, LoadAndForward) {
```

Comment:
It may be confused to aten/core. Could we use "RuntimeTest"?

Reply:
Updated.